docs/src/api/websocket/_blockSubscribe.mdx

```diff
@@ -52,35 +52,71 @@ Configuration object containing the following fields:
   name="commitment"
   type="string"
   optional={true}
+  defaultValue={"finalized"}
   href="/api/http#configuring-state-commitment"
-></Field>
-
-<Field name="transactionDetails" default="full" type="string" optional={true}>
-  level of transaction detail to return, either "full", "signatures", or "none".
+>
+  <li>
+    the default is <code>finalized</code>
+  </li>
+  <li>
+    <code>processed</code> is not supported.
+  </li>
 </Field>
 
-<Field name="showRewards" type="bool" defaultValue="true" optional={true}>
-  whether to populate the `rewards` array.
+<Field name="encoding" type="string" optional={true} defaultValue={"json"} href="/api/http#parsed-responses">
+
+encoding format for each returned Transaction
+
+<Values values={["json", "jsonParsed", "base58", "base64"]} />
+
+<details>
+
+- `jsonParsed` attempts to use program-specific instruction parsers to return
+  more human-readable and explicit data in the `transaction.message.instructions` list.
+- If `jsonParsed` is requested but a parser cannot be found, the instruction
+  falls back to regular JSON encoding (`accounts`, `data`, and `programIdIndex` fields).
+
+</details>
+
 </Field>
 
-<Field name="encoding" type="string" defaultValue="base64" optional={true} href="/api/http#parsed-responses">
+<Field name="transactionDetails" type="string" optional={true} defaultValue={"full"}>
+
+level of transaction detail to return
 
-Encoding format for Account data
+<Values values={["full", "accounts", "signatures", "none"]} />
 
-<Values values={["base58", "base64", "base64+zstd", "jsonParsed"]} />
+<details>
+
+- If `accounts` are requested, transaction details only include signatures and
+  an annotated list of accounts in each transaction.
+- Transaction metadata is limited to only: fee, err, pre_balances,
+  post_balances, pre_token_balances, and post_token_balances.
+
+</details>
+
+</Field>
+
+<Field name="maxSupportedTransactionVersion" type="number" optional={true}>
+
+the max transaction version to return in responses.
 
 <details>
 
-- `base58` is slow
-- `jsonParsed` encoding attempts to use program-specific state parsers to return
-  more human-readable and explicit account state data.
-- If `jsonParsed` is requested but a parser cannot be found, the field falls back
-  to `base64` encoding, detectable when the `data` field is type `string`.
+- If the requested block contains a transaction with a higher version, an
+  error will be returned.
+- If this parameter is omitted, only legacy transactions will be returned, and
+  a block containing any versioned transaction will prompt the error.
 
 </details>
 
 </Field>
 
+<Field name="rewards" type="bool" optional={true}>
+  whether to populate the `rewards` array. If parameter not provided, the
+  default includes rewards.
+</Field>
```

Comment:
Not sure if this lines up 100%, but in the rust code, this is `show_rewards`:
https://github.com/solana-labs/solana/blob/21b2fce6f47899eb3f8d6be1f18431c808349811/rpc-client-api/src/config.rs#L211

Reply:
Yep I think I missed this when I copied the parameters from HTTP. Will change

Reply:
Hmm, on the diff, I still see `rewards`; were you intending to change this to `showRewards` ?

Reply:
Ahh sorry updated the wrong one by accident. Done!